packages/tasks/src/tasks/image-feature-extraction/data.ts

```diff
@@ -0,0 +1,52 @@
+import type { TaskDataCustom } from "..";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description:
+				"Wikipedia dataset containing cleaned articles of all languages. Can be used to train `feature-extraction` models.",
```

Comment:
TODO?

Reply:
ah yes I wanted to find an image pretraining dataset